designsafe/apps/workspace/templates/designsafe/apps/workspace/app_variant_plugin.html

```diff
@@ -1,3 +1,11 @@
+{% if listing|length == 1 %}
+<section class="s-app-version-list">
+    <h2>Already know everything?</h2>
+    {% for variant in listing %}
+    <a class="btn btn-success" href="{{variant.href}}">Get Started</a>
+    {% endfor %}
```

Comment:
Should I instead 'render first variant' e.g. `{% with listing |first as variant %}` ([source](https://stackoverflow.com/questions/1479206/django-templating-how-to-access-properties-of-the-first-item-in-a-list); alternatives exist)?

Reply:
Either way imo